release/example/crk.wordlist_wahkohtowin/source/example.crk.wordlist_wahkohtowin.model.kps

```diff
@@ -15,7 +15,7 @@
   </Info>
   <Files>
     <File>
-      <Name>..\build\example.crk.wordlist_wahkohtowin.model.js</Name>
```

Comment:
According to the .kps spec, these should be backslashes. This is not ideal, but it is what it is. The compiler needs to transform them instead.

Reply:
alright, I added a regex for the slashes